libs/blocks/aside/aside.css

```diff
@@ -41,11 +41,11 @@
   gap: var(--spacing-l);
 }
 
-.aside:not(.notification):not(.inline) .foreground> :first-child {
+.aside:not(.notification):not(.inline) .foreground >  :first-child {
```

Comment:
L44, L48, L506 have double spaces after the `>`, they could be removed.

Reply:
I will be sure to fix those 🙏🏻